tools/packaging/packaging.psm1

```diff
@@ -3038,12 +3038,16 @@ Create a smaller framework dependent package based off fxdependent package for d
 
 .PARAMETER Path
 Path to the folder containing the fxdependent package.
+
+.PARAMETER IsWindows
+Specify this switch if the Windows runtimes are to be kept.
 #>
 function ReduceFxDependentPackage
 {
     [CmdletBinding()]
     param(
-        [Parameter(Mandatory)] [string] $Path
+        [Parameter(Mandatory)] [string] $Path,
+        [switch] $IsWindows
```

Comment:
Can we use a name that is different than the built in variable?   This seems like it would be very confusing in the future.

Reply:
Fixed.